splunk_add_on_ucc_framework/commands/openapi_generator/ucc_object.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from typing import List
+from splunk_add_on_ucc_framework.commands.openapi_generator.json_to_object import DataClasses
+
+class GlobalConfig(DataClasses):
+    def _validate_actions_list(self, *, actions: List[str]) -> None:
+        ALLOWED_VALUES = {"edit", "delete", "clone"}
+        for action in actions:
+            if action not in ALLOWED_VALUES:
+                raise ValueError(f"Action {action} not in the list of allowed values ({ALLOWED_VALUES})")
+    
+    def _validate_configuration_tabs(self,*,tabs: List[DataClasses]=None) -> None:
+        tab_names = set()
+        for tab in tabs:
+            if tab.name not in ['logging','proxy'] and hasattr(tab, "table") and hasattr(tab.table, "actions"):
+                self._validate_actions_list(actions=tab.table.actions)
+            if tab.name in tab_names:
+                raise Exception(f'globalConfig.json contains duplicated Configuration tab {tab.name}')
```

Comment:
I think this needs to be validated during the globalConfig validation phase, not here. Otherwise, you will have magic logic here which may eventually be not consistent with the changes we are doing to the globalConfig validator class.

Reply:
We are already validating this in schema.